sandbox/settings.py

```diff
@@ -201,6 +201,14 @@ class Base(DRFMixin, MagnifyCoreConfigurationMixin, Configuration):
         "dockerflow.django.middleware.DockerflowMiddleware",
     )
 
+    # Swagger
+    API_URL = values.Value("http://localhost:8071")
```

Comment:
I think you should not put this default value in Base settings but instead you can force its value to "http://localhost:8071" in the Development settings below.

Reply:
from what I see you are overwriting the API_URL value just after creating it so it's the same as before. Did you trying what I suggested and is there a difficulty that I overlooked?

Reply:
Yup it wasn't working anyway ^^'. I tried another way around